vault_contract/src/storage.rs

```diff
@@ -0,0 +1,81 @@
+use crate::{did::Did, error::ContractError, issuer::Issuer};
+use soroban_sdk::{contracttype, panic_with_error, Address, Env, Map, String, Vec};
+
+#[derive(Clone)]
+#[contracttype]
+pub enum DataKey {
+    Admin,   // Address
+    Dids,    // String
+    Issuers, // Address
+}
```

Comment:
@juanmagiraldor since we're going to manage issuer authorization per DID. IMO we could define the data key `Issuers(String)`. This key stores a `Map<Address, Issuer>`.